server/app/services/cloud/aws/SimpleStorage.java

```diff
@@ -207,7 +209,16 @@ public S3Presigner getPresigner() {
 
     @Override
     public String bucketAddress() {
-      return String.join("/", localEndpoint, bucket);
+      try {
+        return S3EndpointProvider.defaultProvider()
```

Comment:
I did this update to be consistent with some other URL which is generated on line 202 `presigner = S3Presigner.builder().endpointOverride(localUri).region(region).build();` That other url uses hosted-style so I updated bucketAddress here to use it as well.